Packs/EDL/Integrations/EDL/README.md

```diff
@@ -172,29 +172,33 @@ In order to get the list of all available fields to search by, you can configure
 
 ### Access the Export Indicators Service by Instance Name (HTTPS) - For Cortex XSOAR 6.x only
 
-**Note:** By default, the route is open without security hardening and might expose you to network risks. Cortex XSOAR recommends that you use credentials to connect to the integration.
+**Note:**  
+By default, the route is open without security hardening and might expose you to network risks. Cortex XSOAR recommends that you use credentials to connect to the integration.
 
 
 To access the Export Indicators service by instance name, make sure ***Instance execute external*** is enabled.
 
 1. Navigate to **Settings > About > Troubleshooting**.
 2.  In the **Server Configuration** section, verify that the ***instance.execute.external*** key is set to *true*. If this key does not exist, click **+ Add Server Configuration** and add the *instance.execute.external* and set the value to *true*. See [this documentation](https://xsoar.pan.dev/docs/reference/articles/long-running-invoke) for further information.
 3. In a web browser, go to:
-    `https://*<xsoar_address>*/instance/execute/*<instance_name>*`
- 
-
-### Access EDLs on Cortex XSOAR 8 and Cortex XSIAM
-
-For Cortex XSOAR 8 and Cortex XSIAM, you can only access the Export Indicators Service using a third-party tool such as cURL.
-- On a tenant, use https://ext-<cortex-xsoar-address\>/xsoar/instance/execute/\<instance-name\>
```

Comment:
Need to make sure that https://ext- is not clickable

Reply:
This part was also removed and what was added was formatted so it's not clickable